pkgs/development/tools/omnisharp-roslyn/default.nix

```diff
@@ -25,9 +28,14 @@ stdenv.mkDerivation rec {
   installPhase = ''
     mkdir -p $out/bin
     cd ..
-		cp -r src $out/
-    ls -al $out/src
-    makeWrapper ${mono5}/bin/mono $out/bin/omnisharp \
+    cp -r src $out/
+    rm -rf $out/src/.msbuild
+    cp -r ${msbuild}/lib/mono/msbuild $out/src/.msbuild
```

Comment:
Can we use ``ln -s`` here?

Reply:
I don't think so, because on the line below, we need to rename the folder from bin to BIn